modules/ROOT/pages/7.0-release-notes.adoc

```diff
@@ -549,81 +566,70 @@ NOTE: The new `trigger` option can handle multiple character strings as the trig
 
 For more information, visit the updated xref:autocompleter.adoc[Autocompleter] documentation.
 
-=== The open-source `template` plugin has been removed.
-// #TINY-10654
+=== Removed `closeButton` from `NotificationSpec`, close button in notification is now rendered by default.
```

Comment:
@ShiridiGandham can you please review this newly added release note entry for TINY-10646?
See preview here: http://docs-feature-70-doc-2327.staging.tiny.cloud/docs/tinymce/latest/7.0-release-notes/#removed-closebutton-from-notificationspec-close-button-in-notification-is-now-rendered-by-default

Reply:
- [ ] TINY-10646 reviewed by @ShiridiGandham

Reply:
LGTM